.Documentation/articles/developer/11-text-to-speech.md

```diff
@@ -13,49 +13,38 @@ Supported text to speech engines:
+ 
+Handling multiple languages can be done by using the `Localization` feature. The different translations are stored as key value pairs within a json file. The `Play TextToSpeech Audio` behavior will use the text matching the localization key. The translation files have to be place inside `StreamingAsset/YourCourse/Localization/` folder. When using the Innoactive Template the naming scheme for these files should be the first two letters of the iso country codes for example EN.json, DE.json or ES.json
 
-The TTS engine which is used, is declared in the definition of a current training configuration. The `RuntimeConfiguration` looks up for a TTS engine declared in `TextToSpeechConfig` config which should be placed at the configs directory. You may take a look at the `AdvancedTrainingController` in our Innoactive Template to see an example of how to implementent text to speech generation for multiple languages.
-
-To setup your text to speech engine you have to do following steps:
-
-1. Provide a valid `TextToSpeechConfig` with your current implementation of your `IRuntimeConfiguration`. If you use the `DefaultRuntimeConfiguration`, it's enough to create a _text-to-speech-config.json_ in your config folder. If you implement your own version of the `IRuntimeConfiguration`, the easiest way to get a `TextToSpeechConfig` is to load it.
-    
-    How to load of a `TextToSpeechConfig`:
-    ```csharp
-    TextToSpeechConfig config = new TextToSpeechConfig();
-    if (config.Exists())
-    {
-        textToSpeechConfig = config.Load();
-    }
-    ```
-2. Provide an `InstructionPlayer` in your `IRuntimeDefinition` which is an AudioSource and will be source of the spoken text.
-3. Add a `Play TTS Audio` behavior to your step and configure it with a fitting localization key.
-4. Use `Localization` to load your dictionary with all localization keys needed. This dictionary is a json file containing only a list of "key" : "text" values. If you want to be able to change these values during runtime, put those language files into your StreamingAssets folder. 
-
-    Example:
-    ```json
-    {
-        "grab_sphere": "Please, grab the sphere using the side button of your controller.",
-        "put_sphere": "Please, move the sphere closer to the cube.",
-        "move_cube": "Behold! The mighty flying cube!",
-        "training_complete": "Congratulations! The training is complete."
-    }
-    ```
+Example EN.json file:
+```json
+{
+    "grab_sphere": "Please, grab the sphere using the side button of your controller.",
+    "put_sphere": "Please, move the sphere closer to the cube.",
+    "move_cube": "Behold! The mighty flying cube!",
+    "training_complete": "Congratulations! The training is complete."
+}
+```
+
+[![Play TextToSpeech Audio example](../images/developer/play-text-to-speech-example.png)](../images/developer/play-text-to-speech-example.png "A play audio behavior configured to use a localization key")
```

Comment:
Is this correct?

Reply:
i guess so, i just copied it from somewhere else, link is correct at least :P